sql/pre_install/tables.sql

```diff
@@ -47,13 +47,13 @@ CREATE TABLE IF NOT EXISTS _timescaledb_catalog.hypertable (
   chunk_sizing_func_schema name NOT NULL,
   chunk_sizing_func_name name NOT NULL,
   chunk_target_size bigint NOT NULL CHECK (chunk_target_size >= 0), -- size in bytes
-  compressed boolean NOT NULL DEFAULT FALSE,
+  compression_state smallint NOT NULL DEFAULT 0,
```

Comment:
Maybe add CHECK  (compression_state >= 0 and AND compresion_state < 3), or is this covered by CHECK constraint below? And, perhaps add explanation of states.

Reply:
Yes, it is covered by the check below.

Reply:
FWIW, we could get away with a char for compression_state. A char is often used by postgresql itself for similar things in its catalog tables.